### PR TITLE
feat(filters): implement min and max uploader count filtering

### PR DESCRIPTION
**Pull request by luckyboy**
This PR implement feature for min seeders and max seeders filtering.
Seeders count filtering will improve share rate for torrents have less seeders.

### CODE REVIEW

File: internal/database/sqlite.go [hunk 1]
```diff
@@ -105,6 +105,8 @@ func (db *DB) migrateSQLite() error {
 		return errors.Wrap(err, "failed to query schema version")
 	}
 
+	db.log.Info().Msgf("PRAGMA user_version %d", version)
```
Comment (KyleSanderson) [inline internal/database/sqlite.go:108]: snuck-in perhaps?

**Review by KyleSanderson — COMMENTED** [review]
Neat! Doesn't look like freeleech time is wired up yet, but off to a very solid start.

File: internal/database/sqlite.go (hunk 1 above)
Comment (luckyboy) [reply internal/database/sqlite.go:108]: Ha, it's indeed snuck-in.

**Comment by luckyboy** [comment]
> Neat! Doesn't look like freeleech time is wired up yet, but off to a very solid start.

Freeleech time is defined by some site, i can't find a way to extract those information from current torznab implement. Maybe i should remove this part from this PR.

File: web/src/screens/filters/sections/Advanced.tsx [hunk 2]
```diff
@@ -252,6 +252,28 @@ const Uploaders = ({ values }: ValueConsumer) => (
         </div>
       }
     />
+    <Input.NumberField
+      name="min_uploader_count"
+      label="Min Uploaders Count"
+      placeholder="Takes any number (0 is infinite)"
+      tooltip={
+        <div>
+          <p>Number of max seeders as specified by the respective unit.</p>
```
Comment (s0up4200) [inline web/src/screens/filters/sections/Advanced.tsx:261]:
```suggestion
          <p>Number of min seeders as specified by the respective unit.</p>
```

**Review by s0up4200 — CHANGES_REQUESTED** [review]

**Review by zze0s — COMMENTED** [review]
Good stuff!!

Some suggestions:

* I suggest changing `uploader` to `seeder` for db and structs
* Add min/max leecher filters like seeder already have
* Add a new filter tab/page called Feeds where these Feeds specific settings are grouped

We have recently discussed about moving the Feed Max Age setting from Feed settings to the filter. It probably makes more sense in the filter. I can take care of that in this or a new PR.

**Comment by zze0s** [comment]
No need to close the PR. If you push to the same branch they will show up here.

**Comment by luckyboy** [comment]
> No need to close the PR. If you push to the same branch they will show up here.

Thank you, i'm still adapting to github. i will try to improve this PR anyway.

**Comment by luckyboy** [comment]
Created a new PR #1342, leechers filtering is included. And the filter settings for seeders and leechers ware arranged to RSS/Torznab/Newznab-specific(FeedSpecific).

**Pull request closed without merging**